### PR TITLE
ISPN-13770 move to Infinispan 14

### PR DESCRIPTION
**Pull request by karesti**
https://issues.redhat.com/browse/ISPN-13770

### CODE REVIEW

File: infinispan-remote/persistence/README.md [hunk 1]
```diff
@@ -1,18 +1,6 @@
-# Use H2 driver with Docker or Podman
```
[the remark this answers is not in the document]
Comment (karesti) [reply infinispan-remote/persistence/README.md:1]: yes I should  be reworking this

**Comment by pruivo** [comment]
@karesti can you update for 14.0.0.Dev03?

**Comment by karesti** [comment]
@pruivo yes

**Comment by pruivo** [comment]
merged! thanks @karesti !

**Pull request merged**